examples.py

```diff
@@ -16,488 +16,317 @@
 Then just run this module as a script.
 """
 
-import os, time
+import os
 import numpy as np
 import pandas as pd
-import warnings
 
 from dbdicom import Folder
+from mdreg import MDReg
+import mdreg.models as mdl
 
-from MDR.MDR import model_driven_registration
-from MDR.Tools import read_elastix_model_parameters, export_results
-
-import models_signal.two_compartment_filtration_model_DCE as DCE
-import models_signal.constant as constant
-import models_signal.DTI as DTI
-import models_signal.DWI_monoexponential as DWI_monoexponential
-import models_signal.DWI_monoexponential_simple as DWI_monoexponential_simple
-import models_signal.T1 as T1
-import models_signal.T1_simple as T1_simple
-import models_signal.T2 as T2
-import models_signal.T2_simple as T2_simple
-import models_signal.T2star as T2star
-import models_signal.T2star_simple as T2star_simple
-
-# To read and write from/to other locations, change these path names
 data = os.path.join(os.path.dirname(__file__), 'data')
 results = os.path.join(os.path.dirname(__file__), 'results')
-# Check if `data` folder has more files besides README.md
-if len([files for path, subdirs, files in os.walk(data)]) <= 1: warnings.warn("Data folder is empty!")
+elastix_pars = os.path.join(os.path.dirname(__file__), 'elastix')
 
+# To read and write from/to other locations, set these path names
```

Comment:
Just a warning that these are your local paths and although it's ok for your local tests, it's not the best way to share examples publicly. Maybe also considering adding a check of folder `data` being empty or not when running these examples.

Something like
```
# Check if `data` folder has more files besides README.md
if len([files for path, subdirs, files in os.walk(data)]) <= 1: warnings.warn("Data folder is empty!")
```